optimizely/config/datafileprojectconfig/config.go

```diff
@@ -39,6 +40,7 @@ type DatafileProjectConfig struct {
 	experimentKeyToIDMap map[string]string
 	experimentMap        map[string]entities.Experiment
 	featureMap           map[string]entities.Feature
+	featureFlagMap       map[string]datafileEntities.FeatureFlag
```

Comment:
See comment at bottom. This is incorrect. We should be using the entities.* which are public facing. The datafileEntities are internal models that we don't expose and are only needed so we can parse into entities.* models